#472 generate rotated thumbnail

Botz opened the pull request:
Fixes #472

arturi commented:
This is great! Thank you! We’ll try to review soon.

arturi commented:
Tested with multiple different images and screenshots from iOS — delightfully correct image orientation each time 🚀 

Would you consider adding a test for the orientation functionality?

Will also ask for a look from @goto-bus-stop.

Botz commented:
Sure, i will try to add test for it.

Botz commented:
I have changed the exif dependency because exifreader isn't IE11 compatible. Exif-Js is smaller and the code looks cleaner.

I also added tests for the rotateImage method

arturi commented:
Thank you again! And sorry for the delay. Merged!

hgupixum commented:
Thanks @Botz 

I see it's merged but I can't upgrade it in my project. When it will be released?

arturi commented:
@hgupixum hi! we just did an npm release recently, before this was merged, and the next one in likely coming in a few weeks, end of the month. Unless there’s a hot bug fix we need to push out, we do releases with some intervals.
